src/styles/blog.module.css

```diff
@@ -0,0 +1,66 @@
+.banner {
+  background: url('../../public/images/srkw2-36.jpg');
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-position: 50% 55%;
+}
+
+.overlay {
+  height: 88vh;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  background-color: rgba(5, 28, 90, 0.3);
+}
+
+.bannerText {
+  color: #fff;
+  font-size: 160px;
+  margin-bottom: 20px;
+}
+
+.bannerIcon {
+}
+
+.blogList {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.blogItem {
+  width: 48%;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  margin: 20px 0;
+}
+
+.blogImage {
+  width: 30%;
+  height: 20%;
+  margin: 0 20px;
+}
+
+.blogItemBody {
+  display: flex;
+  flex-direction: column;
+  padding: 0 20px;
+}
+
+.blogItemTitle {
+  font-size: 22px;
+}
+
+.blogItemMetadata {
+  font-size: 13px;
+  font-weight: 600;
+  text-transform: uppercase;
+  color: #999999;
+}
+
+.blogItemSummary {
+  font-size: 16px;
+  font-weight: 500;
+}
```

Comment:
We're trying to move off of CSS modules so it would be best to rewrite this using MUI components, `sx` prop, and `styled()` (see #28 for more details)